memory_core/genesis_new/doublebuffer_control.svp

```diff
@@ -92,14 +92,14 @@ output logic [`$awidth-1`:0]  doublebuffer_addr_mem [`$bbanks-1`:0];
 input logic  [`$wwidth-1`:0]  doublebuffer_data_out [`$bbanks-1`:0];
 
 
-input logic [12:0]            depth;
+input logic [15:0]            depth;
```

Comment:
Why increase the size of the `depth`?

Reply:
since we have 32 rows of tiles and each tile supports 1K words, we can chain together up to 32k of memory (I don't think one would do this, but it should be supported), so that gets us to 15 bits, but 16 bits looks nicer/matches every if im being honest.